ui/src/components/SampleView/SampleView.css

```diff
@@ -176,3 +176,7 @@
   color: #000;
   font-weight: bold;
 }
+
+.hid-by-user {
+  background-color: #aaa !important;
```

Comment:
Isn't it usually bad practice to use `!important` in CSS?

Reply:
You're right. I switched to using data attributes to override bootstrap's default css.